OpenRA.Mods.Common/Warheads/FireClusterWarhead.cs

```diff
@@ -52,23 +52,22 @@ public override void DoImpact(Target target, WarheadArgs args)
 
 			var map = firedBy.World.Map;
 			var targetCell = map.CellContaining(target.CenterPosition);
-			var damageModifiers = args.DamageModifiers;
 
 			var targetCells = CellsMatching(targetCell, false);
 			foreach (var c in targetCells)
-				FireProjectileAtCell(map, firedBy, target, c, damageModifiers);
+				FireProjectileAtCell(map, firedBy, target, c, args);
 
 			if (RandomClusterCount != 0)
 			{
 				var randomTargetCells = CellsMatching(targetCell, true);
 				var clusterCount = RandomClusterCount < 0 ? randomTargetCells.Count() : RandomClusterCount;
 				if (randomTargetCells.Any())
 					for (var i = 0; i < clusterCount; i++)
-						FireProjectileAtCell(map, firedBy, target, randomTargetCells.Random(firedBy.World.SharedRandom), damageModifiers);
+						FireProjectileAtCell(map, firedBy, target, randomTargetCells.Random(firedBy.World.SharedRandom), args);
 			}
 		}
 
-		void FireProjectileAtCell(Map map, Actor firedBy, Target target, CPos targetCell, IEnumerable<int> damageModifiers)
+		void FireProjectileAtCell(Map map, Actor firedBy, Target target, CPos targetCell, WarheadArgs whArgs)
```

Comment:
Style nit: this is part of the public API, so IMO it would be cleaner to call this one `args` then use `projectileArgs` below.